packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -164,6 +174,12 @@ abstract class FlameChartState<T extends FlameChart, V> extends State<T>
       upperBound: FlameChart.maxZoomLevel,
       vsync: this,
     )..addListener(_handleZoomControllerValueUpdate);
+
+    verticalExtentDelegate = FixedExtentDelegate(
```

Comment:
this is actually a good use for fixed extent builder even if it wasn't handly to layer this functionality on the ExtentDelegateList.